.rubocop.yml

```diff
@@ -31,6 +31,9 @@ RSpec/MultipleExpectations:
 RSpec/SubjectStub:
   Enabled: false
 
+RSpec/FilePath:
+  Enabled: false
+
```

Comment:
If `rspec-rubocop` thinks `Fail2Ban` doesn't translate to `fail_2_ban_spec.rb`, then we can do a specific workaround here with [`CustomTransform`](https://github.com/rubocop-hq/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/file_path.rb#L14):

```yaml
RSpec/FilePath:
  CustomTransform:
    - 'Fail2Ban'
```

Reply:
Wow that's cool, I shouldn't have assumed there was no way to fix that. Will fix shortly. I'll give it 24 hours before I merge the next pull request next time, sorry haha!

Reply:
I just happened to be in the "we can do better" mindset. I didn't know about this before :)

Oh, no worries: I'll try to get a look at everything you can throw at me as soon as I can (probably in less than 24 hours :smile:). If you want, feel free to request a review from me!

Reply:
Will do next time!

Fixed in https://github.com/taylorthurlow/panda-motd/commit/1723d1bc85c4d3631305ba3dbca241afc65d56dd.